Modules/Scripted/DICOMLib/Resources/DICOM/dcmtk/storescu.cfg

```diff
@@ -0,0 +1,62 @@
+#
+#  Copyright (C) 2003-2021, 3DSlicer
+#
+#  Author:  Rob Oakes
+#
+#  Purpose: Storescu configuration optimized for DICOM-SR and DICOM-SEG files
```

Comment:
The file name is very generic and in contrast the purpose is very specific. We could either add support for multiple configuration files (each with very name and content very specific for a certain use case) or add just one generic configuration file (with a generic name, which would contain all the customizations that we want to make in Slicer). I think the latter is somewhat simpler and we can switch to using a list of configuration files if we find that we cannot fulfill all use cases with a single configuration file.

Reply:
Agreed. I'll adopt this approach into the merge request when I revise it tomorrow.

Reply:
The file has been renamed to `storescu-seg.cfg`, and the description has been greatly expanded to provide information about why it is needed, how to extend, and provide the URL to the source file in the DCMTK source code (GitHub).